sound/soc/intel/boards/sof_sdw.c

```diff
@@ -617,6 +617,15 @@ static const struct dmi_system_id sof_sdw_quirk_table[] = {
 		},
 		.driver_data = (void *)(SOC_SDW_CODEC_SPKR),
 	},
+	/* Pantherlake devices*/
+	{
+		.callback = sof_sdw_quirk_cb,
+		.matches = {
+			DMI_MATCH(DMI_PRODUCT_FAMILY, "Intel_ptlrvp"),
+		},
+		.driver_data = (void *)(SOF_SDW_TGL_HDMI |
+					SOC_SDW_PCH_DMIC),
```

Comment:
Also can you clarify why you need the PCH_DMIC? Is this a supported configuration? At the very least there should be an explanation of which microphone path is supported.

Reply:
hi Pierre,
This is the POR configuration.

Reply:
Humm, now I am completely lost. You list rt5682 in this PR, along with RT711. Neither are by default on the RVP, so this quirk is problematic indeed.

Reply:
Hi Pierre,
RT722 is the onboard codec and the patch adds RT722 to the ACPI table.
sorry for the wrong branch name.